zeroconf.py

```diff
@@ -1949,17 +1959,22 @@ def create_sockets(
     interfaces = normalize_interface_choice(interfaces, ip_version)
 
     respond_sockets = []
+    respond_socket_options = []
 
     for i in interfaces:
+        respond_socket = None
+        respond_socket_optopn = None
```

Comment:
Typo here.

Also, it's mildly inconvenient that we have a singular version of a plural name in the same code block. Maybe call it `multicast_option` or something?